examples/advanced-project/package.json

```diff
@@ -11,6 +11,7 @@
   "license": "ISC",
   "devDependencies": {
     "checkly": "latest",
-    "jiti": "^2"
+    "jiti": "^2",
+    "@playwright/test": "1.55.1"
```

Comment:
Can we at least change this to `^1.55.1` to make the version more flexible? Or just `^1` like we do for `jiti`.